packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +6,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
 
 1. Create python file
 2. Add PySpark code to the python file.
-3. Click the "Run" icon in the tab bar and select "Upload and Run File on Databricks"
-
-This will start the code synchronization and run the active python file on the configured cluster. The result is printed in the "debug" output panel.
+3. Click the "Databricks Run" icon in the tab bar and select "Upload and Run File on Databricks"
 
-![run](./images/run.gif)
+This will deploy the code to the selected Databricks workspace and run it on the cluster. The result is printed in the "debug" output panel.
 
 ### <a id="running-code-as-workflows"></a>Running PySpark and notebooks as a Workflow
 
 1. Create a python file or a python based notebook
     1. You can create a python based notebook by exporting a notebook from the Databricks web application or use a notebook that is already tracked in git, such as https://github.com/databricks/notebook-best-practices
-2. Click the "Run" icon in the tab bar and select "Run File as Workflow on Databricks"
+2. Click the "Databricks Run" icon in the tab bar and select "Run File as Workflow on Databricks"
 
 This will run the file using the Jobs API on the configured cluster and render the result in a WebView.
 
-### <a id="run-configurations"></a>Advanced: Running using custom run configurations
+## <a id="dabs"></a>Deploying Databricks Asset Bundle
+
+### <a id="what-is-dab"></a>What is a Databricks Asset Bundle?
+
+Databricks Asset Bundles make it possible to describe Databricks resources such as jobs, pipelines, and notebooks as source files. These source files provide an end-to-end definition of a project, including how it should be structured, tested, and deployed, which makes it easier to collaborate on projects during active development. For more information, see [Databricks Asset Bundle](https://docs.databricks.com/en/dev-tools/bundles/index.html).
+
+### <a id="deploy-dab"></a>Deploying a Databricks Asset Bundle?
+
+1. In the Databricks panel, look for the "Bundle Resource Explorer" view.
+2. Click on the "Deploy" button.
+3. You can monitor the deployment status in the log output window.
+
+### <a id="deploy-run-job-pipeline"></a>Run a Job or Pipeline
+
+You can run a job or a pipeline managed by Databricks Asset Bundles, from the "Bundle Resource Explorer" view.
+
+1. In the Databricks panel, look for the "Bundle Resource Explorer" view.
+2. Hover on the line with the job or pipeline you want to run.
+3. Click on the "Run" button.
+
+This will deploy the Databricks Asset Bundle and run the job or pipeline. You can monitor the run progress in the output terminal window. You can also open the run, job or pipeline in workspace by clicking on the "Open link externally" button.
+
+#### Use interactive cluster for running jobs
+
+By default, a job is run using a jobs cluster. You can change this behaviour and use the interactive cluster selected previously ([Select a cluster](#select-cluster)) to run the job.
```

Comment:
I think it's fine to leave it, but just a callout that American English is "behavior".